service/src/main/java/bio/terra/cbas/dao/MethodDao.java

```diff
@@ -16,8 +16,8 @@ public MethodDao(NamedParameterJdbcTemplate jdbcTemplate) {
 
   public int createMethod(Method method) {
     return jdbcTemplate.update(
-        "insert into method (id, method_url, input_definition, entity_type)"
-            + " values (:id, :methodUrl, :inputDefinition, :entityType)",
+        "insert into method (id, method_url, input_definition, record_type)"
+            + " values (:id, :methodUrl, :inputDefinition, :recordType)",
```

Comment:
I think you also need to update `method` table definition in database by adding a new changeset in `resources/changelog/changesets/`.

Reply:
is this for database migration? if so, are we at the point now where we want to be accumulating these migrations? (genuine question; I'm not sure how this works / what it's for)